src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/Completion/RazorCompletionItemProvider.ts

```diff
@@ -73,45 +73,17 @@ export class RazorCompletionItemProvider
                         const rangeEnd = this.offsetColumn(completionCharacterOffset, hostDocumentPosition.line, range.end);
                         completionItem.range = new vscode.Range(rangeStart, rangeEnd);
                     }
-                } else if ((completionItem as any).range2) {
```

Comment:
I dug through the code and `range2` doesn't seem to be a thing anymore for both `CompletionList` and `CompletionItem` so I don't think we need this check.